_includes/scenarios/integrate/integration-example.md

```diff
@@ -1,151 +1,114 @@
+  - Make sure the [docs are enabled]({% link _docs_operate/configuration.md %}#corehttpapi) for the documentation route to work
   - Get the API key that was configured during installation of the Connector (it needs to be sent in the `X-API-KEY` header of every HTTP request)
-- You need **version 2** of the [enmeshed App]({% link _docs_use/install-the-app.md %}) installed on your mobile device.
+- You need the [enmeshed App]({% link _docs_use/install-the-app.md %}) installed on your mobile device.
 
 ## Establishing Relationships
 
-In order to communicate with another Identity, a Relationship to that Identity is required. In this first part of the tutorial you will learn how to establish a Relationship between your Connector and another Identity. In this case the other Identity will be the App, but it could be another Connector as well.
+In order to communicate with another Identity, a Relationship to that Identity is required.
+Thus, in this first part of the tutorial you will learn how to establish a Relationship between your Connector and another Identity.
+In our example the other Identity will be an App.
+However, it could be another Connector, as well, e.g. if two companies want to exchange data securily without opening a port or network.
+
+The way we will establish the Relationship is via a Relationship Template.
+This is created by the Connector and contains its display name, as well as the data it would like to receive from the other Identity.
+Then, the App user fills out all required information of the template and sends the Relationship Request to the Connector.
+It in turn accepts the Request, which results in the creation of a new Relationship.
 
 ### Connector: Create an Attribute
 
-In order to share an Attribute via a Relationship Template, we need to create one by executing `POST /api/v2/Attributes` with the following payload:
-
-```json
-{
-  "content": {
-    "@type": "IdentityAttribute",
-    "owner": "<your connector's Address>",
-    "value": {
-      "@type": "DisplayName",
-      "value": "Connector Tutorial"
-    }
-  }
-}
-```
-
-You can query the Connector's Address under the route `/api/v2/Account/IdentityInfo`. If you are using the Demo Connector of this Tutorial, the Address is `id134nJmN7E4Carb6KyRJyePVnXxVHEYQgWD`.
-{: .notice--info}
+Firstly, we want to create a display name for our Connector.
+While communicating with the other Identity in the subsequent steps, we will choose to share this display name.
+Technically, for this we need to create an [IdentityAttribute]({% link _docs_integrate/data-model-overview.md %}#identityattribute) with an IdentityAttribute Value of type [DisplayName]({% link _docs_integrate/attribute-values.md %}#displayname) for our Connector.
+To do so, proceed as described in the Create own IdentityAttribute guide with the following content.
 
-{% include rapidoc api_route_regex="^post /api/v2/Attributes$" %}
```

Comment:
why are the snippets removed? this guide should be self-contained, now it's a jump-jump-jump-jump-jump-jump

Reply:
we only wanted to use rapidocs in usecases, not in scenarios

Reply:
(IMO)

Reply:
maybe we have to move this out of the scenarios to make this clear.

Reply:
taking out of scenarios sounds promising, we need to discuss this with @stnmtz